WordPress/Classes/Models/StatsRecord+CoreDataClass.swift

```diff
@@ -42,7 +44,10 @@ public enum StatsRecordType: Int16 {
         case .lastPostInsight, .allTimeStatsInsight, .publicizeConnection, .followers:
 
             return false
-        case .postStats, .blogStats:
+        case .postStats,
```

Comment:
I think this should be resolved in the same way as my comment for #11047. Please. 😄